Sources/RealModule/ApproximateEquality.swift

```diff
@@ -0,0 +1,154 @@
+//===--- ApproximateEquality.swift ----------------------------*- swift -*-===//
+//
+// This source file is part of the Swift Numerics open source project
+//
+// Copyright (c) 2020 Apple Inc. and the Swift Numerics project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+extension Numeric where Magnitude: FloatingPoint {
+  /// Compares `self` and `other` for approximate equality with a specified
+  /// relative tolerance and implicit absolute tolerance.
```

Comment:
I assume the majority of folks will opt for this variant? It'd be nice if its summary doc comment was more approachable than the full `isApproximatelyEqual(to:absoluteTolerance:relativeTolerance:norm:)`

Reply:
Yeah, there's a missing intro paragraph here that says something like "use this if you don't know better" (but spelled out in a more welcoming manner).